test/wallet-test.js

```diff
@@ -1177,6 +1177,47 @@ describe('Wallet', function() {
     assert.strictEqual(t2.getFee(), 10000);
   });
 
+  it('should use the change address in the options', async () => {
```

Comment:
I think this test could follow the same form as the example above for `http-test`.